src/modules/DialogOverlay.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <Dialog
+    v-if="isDialogOpen"
+    id="widget-dialog"
+    :button-close-label="buttonCloseLabel"
+    @toggle-dialog="toggleDialog"
+  >
+    <template #header> <slot name="header"></slot> </template>
```

Comment:
Nit: Looks like the formatting might be a bit off here

Reply:
thank you. done.